core/capabilities.go

```diff
@@ -20,16 +21,21 @@ type ModelConstraint struct {
 type Capability int
 type CapabilityString []uint64
 type Constraints struct {
+	minVersion string
+}
+type PerCapabilityConstraints struct {
 	// Models contains a *ModelConstraint for each supported model ID
 	Models ModelConstraints
 }
-type CapabilityConstraints map[Capability]*Constraints
+type CapabilityConstraints map[Capability]*PerCapabilityConstraints
 type Capabilities struct {
-	bitstring   CapabilityString
-	mandatories CapabilityString
-	constraints CapabilityConstraints
-	capacities  map[Capability]int
-	mutex       sync.Mutex
+	bitstring             CapabilityString
+	mandatories           CapabilityString
+	version               string
+	constraints           Constraints
+	capabilityConstraints CapabilityConstraints
```

Comment:
@victorges could you check if this extra field was added at the right positions at the transcoding side? Thanks 🙏🏻.

Reply:
Not sure I understand what you mean with position

Reply:
Ah sorry for that since the same exact structure is used in this version there is no change for the transcoding side code and it should work. Let's re-discuss when I have your improved message implemented 👍🏻.